tslint.json

```diff
@@ -151,6 +151,8 @@
     "no-switch-case-fall-through": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
+    // Uncomment to following line and run `yarn tslint --fix` to remove unused imports
+    // "no-unused-variable": [true, {"ignore-pattern": "^_"}],
```

Comment:
Why don't we activate it as a standard?

Reply:
Because we have some unused constructor parameter that we can't remove yet and the _ does not seem to work. But I can check it quickly.